arch/arm/boot/dts/ni-bluefinLC.dts

```diff
@@ -0,0 +1,121 @@
+/dts-v1/;
+/include/ "ni-zynq.dtsi"
+
+/* NIDEVCODE 78C7 */
+/* NIDEVCODE 78B9 */
+
+/ {
+	model = "NI Bluefin";
```

Comment:
We just got the code name and PID. Let's update it to
"NI Bluefin LC" with device code 0x7B37 and 0x7B38

Reply:
Updated the model code name and device code.